packages/oui-color-badge/_mixins.less

```diff
@@ -57,15 +61,17 @@
     }
   }
 
-  .color-badge-theme(@name, @color) {
-    &_@{name}::before {
-      background-color: @color;
+  .color-badge-theme(@color-name, @color-value) {
+    &_@{color-name}::before {
+      background-color: @color-value;
     }
   }
 
   .color-badge-themes(@index: length(@oui-color-badge-themes)) when (@index > 0) {
     .color-badge-themes(@index - 1);
-    @name: extract(@oui-color-badge-themes, @index);
-    .color-badge-theme(@name, @@name);
+    @color-name: extract(@oui-color-badge-themes, @index);
+    @color-value: @@color-name;
+
+    .color-badge-theme(@color-name, @color-value);
```

Comment:
Why replacing something that work well as is ?
There is no plus-value, unless readability.

We could remove all the modifications on this file :)

Reply:
Readability is a plus-value

Reply:
There is no need to rename existing variables/mixins, if there is no useful changes. 
It will depend of each point of view.